spec/setup.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Test setup for server-side tests.
+ */
+var chai = require("chai");
+var sinonChai = require("sinon-chai");
+
+// Add chai plugins.
+chai.use(sinonChai);
```

Comment:
I would rather not have dependencies that we aren't using in anyway

On Thu, Apr 23, 2015, 2:17 PM Eric Baer <notifications@github.com> wrote:

> In spec/setup.js
> <https://github.com/patrickkettner/krabby/pull/21#discussion_r28988649>:
>
> > @@ -0,0 +1,16 @@
> > +/**
> > + * Test setup for server-side tests.
> > + */
> > +var chai = require("chai");
> > +var sinonChai = require("sinon-chai");
> > +
> > +// Add chai plugins.
> > +chai.use(sinonChai);
>
> We're not using it. I suppose I could pull it in when we do. I wanted to
> start us out with the "complete" stack but I'm also find to build out as we
> go. Up to you - I'm indifferent.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/patrickkettner/krabby/pull/21/files#r28988649>.
>